### ISSUE TITLE
ability to delete entries from database report screen

### ISSUE THREAD

**Issue by hrez** [keepassxreboot/keepassxc#5062]
## Summary
request ability to delete entries from database report window

## env
KeePassXC - Version 2.6.0 Revision: 0765954

Hi, 
my database is ancient and has a lot of legacy and obsolete entries in it.
The database report for both  healhcheck and HIBP has a long list of entries and many of them can be deleted.
I couldn't find a way to do that from the report (let me know if I am wrong).  I can open an entry to edit but can't delete it from there either.
To be clear, I am not looking to exclude an entry from the report but rather delete it i.e move to recycle bin.
Basically I want delete button on a database report.

**Comment by droidmonkey** [comment]
Duplicate request
